scripts/deploy-production.js

```diff
@@ -90,6 +94,17 @@ const getCommitMessages = () =>
     );
   });
 
+const getCurrentlyDeployedCommit = () =>
+  new Promise((resolve, reject) => {
+    exec("sh scripts/get-current-deployed-commit.sh", (err, stdout) => {
```

Comment:
Should probably invoke this with bash (or use `sh` everywhere)
```suggestion
    exec("bash scripts/get-current-deployed-commit.sh", (err, stdout) => {
```